### PR TITLE
changed the effects of input box and the font family

### LINKED ISSUE #18: Change the font-family and styling the input boxes
enhancing its styling by changing the font-family and styling the input boxes(like increasing some width and adding shadows and radius)
i1
i2
I am a GSSOC'21 participant. Please assign me this issue under GSSOC

### PR DESCRIPTION
**Pull request by gnikita432**
## Related Issue 

- Fixed issue no #18 

Closes: #18 

### Describe the changes you've made

Give a clear description what modifications you have made

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Mention any unusual behaviour of your code (Write NA if not)
Any unusual behaviour of your code

## Additional Info (optional)
Any additional information you want to give

## Checklist:

- [x] My code follows the guidelines of this project.
- [ x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [ x] My changes generate no new warnings.
- [ x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by ankitapuri** [comment]
@rish-singhal , review this PR once

**Comment by rish-singhal** [comment]
seems good to me @ankitapuri

File: index.html [hunk 1]
```diff
@@ -12,6 +12,10 @@
    <link href="css/style.css" rel="stylesheet">
    
    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script&family=Lobster&display=swap" rel="stylesheet">
+   
+  <link href="https://fonts.googleapis.com/css2?family=Dancing+Script&family=Lobster&display=swap" rel="stylesheet">
```
Comment (rish-singhal) [inline index.html]: this is not needed I guess... as Dancing Script is already there.

File: css/style.css [hunk 2]
```diff
@@ -114,8 +116,26 @@ td {
...
+  border-radius: 10px;
+  border: 1px solid #d1d1d1;
+  box-shadow: rgba(50, 50, 93, 0.25) 0px 30px 60px -12px inset, rgba(0, 0, 0, 0.3) 0px 18px 36px -18px inset;
+  font-family: inherit;
+  font-size: 1em;
+  line-height: 1.45;
+  outline: none;
+  padding:0.2em;
+}
+input:hover {
```
Comment (rish-singhal) [inline css/style.css:131]: add empty lines between lines 130, 131 and 133-134 for better readability.

File: css/style.css [hunk 3]
```diff
@@ -1,5 +1,7 @@
 body {
     text-align: center;
+    font-family: 'Lora', serif;
+
```
Comment (rish-singhal) [inline css/style.css]: this empty line can be removed.

**Review by rish-singhal — COMMENTED** [review]
have highlighted some changes, we can merge after they are resolved.

**Comment by gnikita432** [comment]
Thank you @rish-singhal I have made the changes.

**Pull request merged**